client/tests/end_to_end.rs

```diff
@@ -113,6 +113,48 @@ async fn register_org() {
     assert!(org.projects.is_empty());
 }
 
+#[async_std::test]
+#[serial]
+async fn register_user() {
+    let _ = env_logger::try_init();
+    let node_host = url::Host::parse("127.0.0.1").unwrap();
+    let client = Client::create_with_executor(node_host).await.unwrap();
+    // Must be distinct sender from other user registrations in tests to avoid
+    // AccountUserAssociated errors.
+    let sender = ed25519::Pair::from_string("//alice", None).unwrap();
```

Comment:
```suggestion
    let sender = ed25519::Pair::from_string("//Alice", None).unwrap();
```
This works now but is going to fail once we introduce fees and “alice” does not have any funds.